contracts/RiskManagerV1.sol

```diff
@@ -139,6 +139,9 @@ contract RiskManagerV1 is IRiskManagerV1, Auctioneer, Ownable {
     ///         assuming enough surplus will accumulate up to that point.
     uint256 public tbtcSurplus;
 
+    /// @notice TODO: add description
+    address public councilMultisig;
```

Comment:
I would introduce another RiskManager contract for that. As far as I recall, the coverage pool can have multiple risk managers approved.

Also, T coverage pool will not cover TBTC v1 liquidations.

Reply:
Yes, the coverage pool can have multiple risk managers approved. Added a new `RiskManagerV2`.